.taskcluster.yml

```diff
@@ -6,8 +6,8 @@ policy:
 tasks:
```

Comment:
This changes to this file are temporary, pending the associated code landing in https://hg.mozilla.org/ci/taskgraph